src/main/assembly/dist/install/db-tables.sql

```diff
@@ -2,7 +2,8 @@ CREATE TABLE Deposit (
     depositId CHAR(36) NOT NULL PRIMARY KEY,
     bagName TEXT,
     creationTimestamp TIMESTAMP WITH TIME ZONE NOT NULL,
-    depositorId VARCHAR(64) NOT NULL
+    depositorId VARCHAR(64) NOT NULL,
+    origin VARCHAR(64) NOT NULL
```

Comment:
Now that I think of it, isn't `VARCHAR(64)` a bit much? The values (for now) are only `SWORD2`, `SMD` or `API`. Perhaps a smaller space should suffice? LMKWYT